src/__tests__/preset.test.js

```diff
@@ -128,4 +128,25 @@ describe("index", () => {
                   });"
             `);
     });
+
+    test("wildcard-import.js", () => {
+        const input = fs.readFileSync("./example/wildcard-import.js");
+        const output = transform(input).code;
+
+        expect(output).toMatchInlineSnapshot(`
+            "\\"use strict\\";
+
+            var DefaultFunction = _interopRequireWildcard(require(\\"./default-function.js\\"));
+
+            function _getRequireWildcardCache() { if (typeof WeakMap !== \\"function\\") return null; var cache = new WeakMap(); _getRequireWildcardCache = function () { return cache; }; return cache; }
+
+            function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== \\"object\\" && typeof obj !== \\"function\\") { return { default: obj }; } var cache = _getRequireWildcardCache(); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj.default = obj; if (cache) { cache.set(obj, newObj); } return newObj; }
```

Comment:
The wildcard interop is ridiculous.  Since all of our modules have `exports.__esModule = true` we could drop it altogether.  I'll save that for a future PR though.